src/Uno.UI/UI/Xaml/UIElement.Layout.cs

```diff
@@ -144,13 +144,13 @@ internal enum LayoutFlag : byte
 			/// Means the Arrange is dirty on at least one child of this element
 			/// </summary>
 			ArrangeDirtyPath = 0b0010_0000,
+#endif
 
 			/// <summary>
 			/// Indicates that first arrange has been done on the element and we can use the
 			/// LayoutInformation.GetLayoutSlot() to get previous finalRect
 			/// </summary>
 			FirstArrangeDone = 0b0100_0000,
```

Comment:
You're activating this flag on all platforms, but I fail to see where you are setting it.